config/routes.rb

```diff
@@ -5,5 +5,5 @@
   # events
   get '/events', to: 'events#index'
   get '/events/:urlname/:event_id', to: 'events#show'
-  get '/events/:urlname/:event_id/random', to: 'events#random_attendee'
+  get '/events/:urlname/:event_id/random', to: 'events#random'
```

Comment:
I think since that method gives an attendee insted of an event, the name random_attendee is more suitable giving its context.